src/styles/sidebar/components/button.scss

```diff
@@ -2,73 +2,23 @@
 @use "../../variables" as var;
 
 /**
- * Expected button markup structure:
+ * Button markup structure as generated by the `<Button>` component when
+ * default styles are not overridden:
  *
- * <button class="button <button--icon-only|button--labeled> [button--compact] [button--primary] [is-active]">
- *   [<SvgIcon class="button__icon" />]
+ * <button class="<button--icon-only|button--labeled>">
+ *   [<SvgIcon class="<button--icon-only__icon|button--labeled__icon>" />]
  *   [<span>label</span>]
  * </button>
  */
-.button {
```

Comment:
Before these changes, stuff was sort of randomly split between the `button` SCSS module and the `buttons` mixin. That has been rectified here. This alone was the source of a lot o' headache.

Reply:
👍